src/plugins/kbn_doc_views/public/views/table.js

```diff
@@ -13,7 +13,8 @@ docViewsRegistry.register(function () {
         hit: '=',
         indexPattern: '=',
         filter: '=',
-        columns: '='
+        columns: '=',
+        updateFilterInQuery: '=filter'
```

Comment:
Why do you need `updateFilterInQuery` in query here? Its bound to the same thing as filter?